search.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class search {
+    static Scanner file;
+    static String pattern;
+    public static void main(String[] args) {
+        if(args.length == 2) {
+            pattern = args[0];
+            try {
+                file = new Scanner(new File(args[1]));
+            } catch (FileNotFoundException e) {
+                System.out.println("File not found");
+            }
+            while(file.hasNextLine()) {
+                String sentence = file.nextLine();
+
+                if(sentence.toLowerCase().indexOf(pattern.toLowerCase()) != -1) {
```

Comment:
Kan använda contains istället för indexof

Reply:
Added the change requested!